proposals/lip-0068.md

```diff
@@ -173,7 +173,7 @@ def computeTransactionSignature(sk: PrivateKeyEd25519, trs: Transaction, chainID
     return signEd25519(sk, MESSAGE_TAG_TRANSACTION, chainID, serializedTrs)
 ```
 
-By convention, the `signatures` property of the transaction object that is signed is required to be the empty array and therefore this property is set accordingly. Further note that the function `signEd25519` is defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#signing-and-verifying-with-ed25519).
+By convention, the `signatures` property of the transaction object that is signed is required to be the empty array and therefore this property is set accordingly. Further note that the function `signEd25519` is defined in [LIP 0062](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0062.md#signing-and-verifying-with-ed25519).
```

Comment:
Same comment holds for all links actually; it is optional of course, but I think easier. Also for example for LIP 38, you are using manually the same link three times. By using the proposed approach, you would not need to change all three links in the future in case a new LIP supersedes again those; with the current status, all three links would need to be updated.

Reply:
change here and in LIP 0038